docs/versions/latest/Release.md

```diff
@@ -35,6 +35,19 @@
 [//]: # (Optional But higlhy recommended Specify *NONE* if missing)
 [//]: # (#### Relevant Documentation:)
 
+### Add missing properties to VcComputeResourceConfigSpec and related class definitions in vRO interfaces
+
+#### Previous Behavior
+
+The VcComputeResourceConfigSpec class is missing a few attributes and types of those attributes
+
+#### Current Behavior
+
+The attributes are missing and corresponding types are missing
```

Comment:
Here is a suggestion on how we can describe the current / new behaviour:
```suggestion
The `enableConfigManager`, `maximumHardwareVersionKey` and `desiredSoftwareSpec` attributes are added to the `VcComputeResourceConfigSpec` class interface. Related class interfaces for `VcDesiredSoftwareSpec`, `VcDesiredSoftwareSpecBaseImageSpec`, `VcDesiredSoftwareSpecVendorAddOnSpec` and `VcDesiredSoftwareSpecComponentSpec` are added vCenter plugin interfaces.
```

Reply:
Got it.